cluster/crds/secrets.crossplane.io_storeconfigs.yaml

```diff
@@ -151,146 +151,6 @@ spec:
                 - Vault
                 - Plugin
                 type: string
-              vault:
```

Comment:
We should definitely remember mentioning this in the release notes. Is there any migration guide to switch to ESS?

Reply:
Just created a draft release with a notice of this breaking change. 
As you suggested before, we can maintain the draft notes there to ensure we record notable/breaking changes thought the release cycle.

Reply:
Cool, yes, I was not suggesting to automate the migration, just to point to a guide about how to do that 👍